tutorials/terraform/sqoop/clusters-mysql-data-proc-and-vm.tf

```
@@ -1,25 +1,28 @@
-# Infrastructure for the Yandex Cloud Managed Service for MySQL® cluster.
+# Infrastructure for the Yandex Cloud Managed Service for MySQL® cluster, Data Proc cluster, and Virtual machine
 #
 # RU: https://cloud.yandex.ru/docs/managed-mysql/tutorials/sqoop
+# EN: https://cloud.yandex.ru/en/docs/managed-mysql/tutorials/sqoop
 #
 # Set the configuration of the Managed Service for MySQL® cluster, Managed Service for Data Proc cluster, and Virtual machine:
 locals {
-  folder_id           = ""      # Your folder ID.
-  network_id          = ""      # Network ID for the Managed Service for MySQL® cluster, Data Proc cluster, and VM.
-  subnet_id           = ""      # Subnet ID (enable NAT for this subnet).
-  storage_sa_id       = ""      # Service account ID for creating a bucket in Object Storage.
+  folder_id           = ""      # Your folder ID
+  network_id          = ""      # Network ID for the Managed Service for MySQL® cluster, Data Proc cluster, and VM
+  subnet_id           = ""      # Subnet ID (enable NAT for this subnet)
+  storage_sa_id       = ""      # Service account ID for creating a bucket in Object Storage
   data_proc_sa        = ""      # Set a Data Proc service account name. It must be unique in the folder.
-  my_cluster_version  = "8.0"   # Set the MySQL® version: 5.7 or 8.0.
-  my_cluster_db       = "db1"   # Set a database name.
-  my_cluster_username = "user1" # Set a database owner name.
-  my_cluster_password = ""      # Set a database owner password.
-  vm_image_id         = ""      # Set a public image ID from https://cloud.yandex.com/en/docs/compute/operations/images-with-pre-installed-software/get-list.
+  my_cluster_version  = "8.0"   # Set the MySQL® version: 5.7 or 8.0
+  my_cluster_db       = "db1"   # Set a database name
+  my_cluster_username = "user1" # Set a database owner name
+  my_cluster_password = ""      # Set a database owner password
+  vm_image_id         = ""      # Set a public image ID from https://cloud.yandex.com/en/docs/compute/operations/images-with-pre-installed-software/get-list
   vm_username         = ""      # Set a username for VM. Images with Ubuntu Linux use the username `ubuntu` by default.
```

```suggestion
  vm_username         = ""      # Set a username for VM. Images with Ubuntu Linux use the `ubuntu` username by default.
```

Ниже в подписи ресурсов используется именно такая последовательность. Давай везде сделаем одинаково. Применимо здесь и во втором файле.